private/bufpkg/bufcheck/internal/bufcheckserver/internal/bufcheckserverhandle/breaking.go

```diff
@@ -2313,8 +2313,16 @@ func handleBreakingPackageNoDelete(
 			for i, previousFile := range previousFiles {
 				previousDescriptors[i] = previousFile
 			}
+			// ResponseWriter only expects a single against file name, so we grab the first one
+			// from the previous descriptors. As noted, this is to provide a path for check ignoring
+			// unstable packages.
+			// TODO: Do we need some way to check all files in previousDescriptors?
```

Comment:
We only support one against location in the new response writer, so picked one of the files. This is only for checking unstable packages, added a new TODO to make this a little clearer.